ui/StatusQ/src/typesregistration.cpp

```diff
@@ -79,6 +80,8 @@ void registerStatusQTypes() {
 
     qmlRegisterType<ModelEntry>("StatusQ", 0, 1, "ModelEntry");
     qmlRegisterType<SnapshotObject>("StatusQ", 0, 1, "SnapshotObject");
+    qmlRegisterType<Keychain>("StatusQ", 0, 1, "Keychain");
+    qRegisterMetaType<Keychain::Status>();
```

Comment:
This seems to be not necessary. Using `Q_ENUM` is sufficient to expose enum.

Reply:
@micieslak there's a runtime error otherwise:
```
QObject::connect: Cannot queue arguments of type 'Keychain::Status'
(Make sure 'Keychain::Status' is registered using qRegisterMetaType().)
```
It's enough to expose as `Q_PROPERTY`, but not for signal arguments.

Reply:
Yeah, has to be explicit imo for the queued invokations